shared/actions/fs/index.js

```diff
@@ -135,6 +135,19 @@ const setTlfSyncConfig = (state, action) =>
     })
   )
 
+const loadSettings = (state, action) =>
+  RPCTypes.SimpleFSSimpleFSSettingsRpcPromise()
+    .then(settings =>
+      FsGen.createSettingsLoaded({
+        settings: Constants.makeSettings({
+          spaceAvailableNotificationThreshold: settings.spaceAvailableNotificationThreshold,
+        }),
+      })
+    )
+
+const setSpaceNotificationThreshold = (state, action) =>
+  RPCTypes.SimpleFSSimpleFSSetNotificationThresholdRpcPromise({threshold: action.payload.spaceAvailableNotificationThreshold})
```

Comment:
should probably have `.then(() => FsGen.createSettingsLoad())` here

Reply:
I was debating how to do that, whether we wanted to handle it in the reducer or do a full settings load.

Reply:
I think we should handle it through the RPC. If we do it in the reducer, there's a chance we'll have inconsistent data.

Reply:
I mean whether to handle it via a `.then(() => createSomeNewAction)` and `.catch(createRevertStateAction)`, or do a full settings load.